app/src/main/kotlin/com/simplemobiletools/dialer/helpers/CallManager.kt

```diff
@@ -7,6 +7,7 @@ import android.telecom.Call
 import android.telecom.CallAudioState
 import android.telecom.InCallService
 import android.telecom.VideoProfile
+import android.util.Log
```

Comment:
guess you added it by mistake

Reply:
Oh, I was using Logs to check the code execution as my android studio debug mode with breakpoints kept hanging.

I will remove it now.